src/test/scala/scorex/core/DefaultModifiersCacheSpecification.scala

```diff
@@ -0,0 +1,120 @@
+package scorex.core
+
+import org.scalatest.{Matchers, PropSpec}
+import org.scalatest.prop.{GeneratorDrivenPropertyChecks, PropertyChecks}
+import scorex.core.consensus.{History, HistoryReader, ModifierSemanticValidity, SyncInfo}
+import scorex.core.consensus.History.ModifierIds
+import scorex.core.serialization.Serializer
+import scorex.crypto.hash.Blake2b256
+
+import scala.util.Try
+
+class DefaultModifiersCacheSpecification extends PropSpec
+  with PropertyChecks
+  with GeneratorDrivenPropertyChecks
+  with Matchers {
+
+  property("ignore adding self as a peer") {
```

Comment:
test name looks like copy-paste of different a test. Please provide a correct name, that describes tested properties